source/performance.txt

```diff
@@ -83,27 +90,27 @@ The :guilabel:`Performance` tab displays various stats:
        operations correspond to a given moment in the charts; i.e. as
        you move over the charts, the corresponding slowest operations
        are displayed in the table. To view the details of a slow
-       operation or :ref:`kill the operation <kill-long-ops>`, click on
+       operation or :ref:`stop the operation <kill-long-ops>`, click on
        the operation.
 
 .. _kill-long-ops:
 
-Kill Slow Operations
+Stop Slow Operations
 --------------------
 
 .. important:: Required Permissions
 
-   On deployments which require
-   :manual:`authentication </core/authentication/>`, to kill operations
-   not owned by the user, the user must have access that includes the
-   :authaction:`killop` privilege action.
+   On deployments that require :manual:`authentication
```

Comment:
[suggestion] Consider, 

"To stop operations you don't own, you must have access that includes the killop privilege action."